src/calculateRentalCost.js

```diff
@@ -28,7 +28,21 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentalPrice = 40;
+  const discountTreeDays = 20;
```

Comment:
Typo in variable name. It should be 'discountThreeDays' instead of 'discountTreeDays'.